build.sbt

```
@@ -6,19 +6,22 @@ ThisBuild / developers ++= List(
   tlGitHubDev("archisman-dey", "Archisman Dey"),
   tlGitHubDev("armanbilge", "Arman Bilge"),
   tlGitHubDev("valencik", "Andrew Valencik"),
+  tlGitHubDev("pantShrey", "Shrey Pant"),
 )
 ThisBuild / startYear := Some(2023)
 ThisBuild / tlSonatypeUseLegacyHost := false
+ThisBuild / resolvers += "Sonatype Releases".at(
+  "https://oss.sonatype.org/content/repositories/releases/",
+)
+ThisBuild / crossScalaVersions := Seq("3.3.4", "2.13.16")
 
-ThisBuild / crossScalaVersions := Seq("3.3.0", "2.13.10")
-
-ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("17"))
+ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("17"), JavaSpec.temurin("21"))
```

We can just go straight to 21.

```suggestion
ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("21"))
```